include/oneapi/dpl/pstl/parallel_backend_utils.h

```diff
@@ -301,6 +306,74 @@ __set_symmetric_difference_construct(_ForwardIterator1 __first1, _ForwardIterato
     return __cc_range(__first2, __last2, __result);
 }
 
+template <typename _ValueType, typename _GetNumThreads, typename _GetThreadNum, typename... _Args>
+struct __enumerable_thread_local_storage
+{
+    template <typename... _LocalArgs>
+    __enumerable_thread_local_storage(_LocalArgs&&... __args)
+        : __num_elements(0), __args(std::forward<_LocalArgs>(__args)...)
+    {
+        std::size_t __num_threads = _GetNumThreads{}();
+        __thread_specific_storage.resize(__num_threads);
```

Comment:
May be better to initialize this vector not in constructor body but in the member initialization list passing into vector size?

Reply:
good point, fixed.